Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -116,7 +114,15 @@ internal sealed class PythonLanguageClient : ILanguageClient, ILanguageClientCus
         // Called by Microsoft.VisualStudio.LanguageServer.Client.RemoteLanguageServiceBroker.UpdateClientsWithConfigurationSettingsAsync
         // Used to send LS WorkspaceDidChangeConfiguration notification
         public IEnumerable<string> ConfigurationSections => Enumerable.Repeat("python", 1);
-        public object InitializationOptions { get; private set; }
+        public object InitializationOptions {
+            get {
+                return new {
+                    vsSupport = true,
```

Comment:
undoing this.. thought we needed a way to tell pylance we were VS.. but there is already another attribute being passed.